### ISSUE TITLE
Add python-socketio package

### ISSUE THREAD

**Issue by ThreeVeil939527** [pyodide/pyodide#2670]
## 🐍 Package Request

- Package Name: python-socketio
- Package URL: [](https://pypi.org/project/python-socketio/)

**Comment by ryanking13** [comment]
Could you try `await pyodide.loadPackage("ssl")`, then import it again? We are currently not shipping ssl by default so you have to explicitly load it.

**Comment by hoodmane** [comment]
It probably won't work even then because sockets don't work.

**Comment by ThreeVeil939527** [comment]
I may be forgetting something but when I run `await pyodide.loadPackage("ssl")` it says `AttributeError: module 'pyodide' has no attribute 'loadPackage'`

**Comment by hoodmane** [comment]
You need to do that from Javascript. Or:
```py
import pyodide_js
await pyodide_js.loadPackage("ssl")
```

**Comment by ThreeVeil939527** [comment]
It is now importing but when I try to connect to a socket.io server it complains about the `requests` library not being installed, I installed the `requests` library and imported it but it is still not working.

**Comment by ryanking13** [comment]
Yes, it is a knows issue because we currently don't support sockets due to browser limitations. See #140

**Comment by ThreeVeil939527** [comment]
Would it be possible to perform the request from a javascript thing and then pass it to socketio?  I am still trying to understand the code that is making the request so I might be completely wrong.

**Comment by ryanking13** [comment]
I don't know about `socketio` internals, but you try monkey-patching it.
For example: https://echorand.me/posts/py-script-monkey-patch-requests/

Though I am not sure that it will work as synchronous requests in the browser is highly limited.
